omnibus/files/private-chef-cookbooks/private-chef/templates/default/oc_erchef.config.erb

```diff
@@ -2,304 +2,330 @@
 %% -*- tab-width: 4;erlang-indent-level: 4;indent-tabs-mode: nil -*-
```

Comment:
Not really sure--most of this stuff wasn't indented correctly at all. I will go through and change all the indents to 4, though.